tests/_helpers.py

```diff
@@ -74,3 +76,16 @@ def new_isfile(path: str) -> bool:
 
     with mock.patch(isfile_ref, new=new_isfile) as is_file_mock:
         yield is_file_mock
+
+
+def skipif_import_error(*module_name: str) -> pytest.MarkDecorator:
+    """Create skipif marker for unimportable modules"""
+    for m in module_name:
+        try:
+            __import__(m)
+            skip = False
+        except ImportError:
+            skip = True
+            break
+
+    return pytest.mark.skipif(skip, reason=f"{module_name} dependency not found")
```

Comment:
`module_name` is a tuple, is this what you meant to put in the message?

Reply:
nope :)